geps/gep-2907/index.md

```diff
@@ -75,6 +73,17 @@ flowchart LR
 
 The above diagram depicts these four segments as edges in a graph.
 
+### TLS mode
+
+TLS can be configured with two distinct modes:
+
+* **Terminate**: the TLS connection is instantiated between the frontend and the
+  Gateway. The connection between the Gateway and the backend is either left unencrypted
+  or a new TLS connection between the two entities is instantiated.
```

Comment:
This might be a little more clear:
```suggestion
  Gateway. The connection between the Gateway and the backend left unencrypted
 unless a new TLS connection between the two entities is configured via BackendTLSPolicy.
```